packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -75,12 +75,13 @@ abstract class Gesturer extends BindingBase implements HitTestTarget, HitTestabl
   }
 
   /// Determine which [HitTestTarget] objects are located at a given position.
-  @override
+  @override // from HitTestable
   void hitTest(HitTestResult result, Point position) {
     result.add(new HitTestEntry(this));
   }
 
-  /// Dispatch the given event to the path of the given hit test result
+  /// Dispatch the given event to the path of the given hit test result.
```

Comment:
There's nothing wrong with this but it sounds a bit like the Uniform Plumbing Code :-)
Maybe:
/// Dispatch the event to the hit test result's path.